rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -190,6 +190,8 @@ List<UserJobData> jobsInfoListPost(@HeaderParam("sessionid") String sessionId, L
      *             Include only jobs with a name that starts with jobName (case in-sensitive)
      * @param projectName
      *             Include only jobs with a project name that starts with projectName (case in-sensitive)
+     * @param bucketName
```

Comment:
actually all these filters (jobName, projectName, bucketName and userName) are case sensitive